src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/prisonersearch/PrisonerSearchClient.kt

```diff
@@ -21,15 +21,6 @@ class PrisonerSearchClient(private val prisonerSearchApiWebClient: WebClient) {
       .bodyToMono(Prisoner::class.java)
       .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
       .block()
-
-  fun getPrisoner(bookingId: Long): Prisoner? =
```

Comment:
Moved function into migration client as only really needed there.  When migration is done the migration client will be deleted.

Reply:
I am trying to group everything that is migration related only into migration specific classes.

This should make the cleanup/removal of migration **only** code easier i.e. touching other classes as little as possible so unused code does not get left in.